manuscript/00-foreword.md

```diff
@@ -7,85 +7,82 @@ This is a prerelease version of *The Tao of tmux*. The estimated release date is
+phones out. This was back when phones still had physical "QWERTY" keyboards.
 
-Despite our home computers sleeping or being off, our usernames in the channel
+Despite our home computers being asleep or turned off, our usernames in the IRC channel we frequently visited
 persisted in the chatroom list. Our screens were lit by a kaleidoscope of colors
 on a black background. We ssh'd with ConnectBot into our cloud servers and
-reattached [`screen(1)`](https://en.wikipedia.org/wiki/GNU_Screen). Just as it
-hit 2AM, our Turkish coffee arrived, the `|away` trailing our online nicknames
-one by one fell off.
+reattached by running [`screen(1)`](https://en.wikipedia.org/wiki/GNU_Screen). Just as it
+hit 2AM, our Turkish coffee arrived, the `|away` status indicator trailing our online nicknames disappeared.
 
-Funnily, even though we knew each other by our real names, we sometimes opted to
-call each other by our nicks. It's something about how the personal
-relationships, formed completely through text, persist in real life.
+It was funny that even though we knew each other by our real names, we sometimes opted to
+call each other by our nicks. It's something about how personal
+relationships, formed completely online, persist in real life.
 
 It seemed as if it were orchestrated, but each of us fell into the same ebb and
-flow of operating our lives. No one told us to do it, but inch by inch we
+flow of living our lives. No one told us to do it, but inch by inch we
 incrementally optimized our lifestyles personally and professionally to arrive
 at destinations that felt eerily alike.
 
 Like many things in life, when we act on autopilot, we sometimes arrive at
 similar destinations. This is often completely unplanned.
 
-So when I go off and write a learning book on a computer application, I hope
-to write it for human beings. Not to sell you on tmux, not to make you like
-it, or hate it, but to tell you what it is, how some use it and leave the rest
+So when I go off and write an educational book about a computer application, I hope
+to write it for human beings. I'm not trying to sell you on tmux, not to make you like
+it, or hate it, but to tell you what it is, how some people use it. I'll leave the rest
 up to you.
 
 ## About this book
 
 This book is currently under development. It's available to read for
 [free on the web](https://leanpub.com/the-tao-of-tmux/read).
 
-You do not need a book to use or understand tmux. But if there is a book, let
+You do not need a book to use or understand tmux. But if there is such a book, let
 this be the one.
 
-I don't like to give a slipshod effort. If I'm going to take the time to write a
+I don't like to make a slipshod effort. If I'm going to take the time to write a
 book, I'm giving it 100%.
 
-tmux to me is a tool I find incredible use with. While I don't attach it to my
+tmux to me is a tool I find incredible useful. While I don't attach it to my personal
```

Comment:
And maybe remove "to me" as it's redundant.

"tmux is a tool I find incredibly useful".